src/database/migrations/20230613000210_AddSiteIdToParticipantsTableAndCreateSharingAuditTrailTable.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('participants', (table) => {
+    table.integer('siteId').unique();
+  });
+
+  await knex.schema.createTable('sharingAuditTrails', (table) => {
+    table.increments('id').primary();
+    table.integer('participantId').references('participants.id').onDelete('CASCADE');
+    table.integer('userId').references('users.id').onDelete('CASCADE');
```

Comment:
We can *never* delete an audit entry, so these cascading deletes aren't OK. We probably won't ever delete participants, so that relationship is probably fine. But we might delete users, so I think we should store the email address as well, and not enforce the reference to the users table.